app/src/main/res/navigation/main_nav.xml

```diff
@@ -6,11 +6,9 @@
     android:id="@+id/main_nav_graph.xml"
     app:startDestination="@id/safeBalancesFragment">
 
-    <include app:graph="@navigation/add_safe_nav" />
-
     <action
         android:id="@+id/action_to_add_safe_nav"
```

Comment:
we don't need this global action now. You can navigate directly to addSafeFragment

Reply:
We need it for `NoSafeFragment` . If you checkout the branch and "look for usages" you will find it is still in use.

Reply:
why can't you go from NoSafeFragment to AddSafeFragment?